docs/self-hosting/methods/install-methods-commercial/kubernetes.md

```
@@ -0,0 +1,568 @@
+        --create-namespace \
+        --namespace plane \
+        -f values.yaml \
+        --timeout 10m \
+        --wait \
+        --wait-for-jobs 
+    ```
+iii. If you've purchased a paid plan, [activate your license key](/self-hosting/manage/manage-licenses/activate-pro-and-business#activate-your-license) to unlock premium features.
+
+## Configuration settings
+
+#### License
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)